libraries/rush-lib/src/logic/operations/test/__snapshots__/BuildPlanPlugin.test.ts.snap

```diff
@@ -0,0 +1,736 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`BuildPlanPlugin build plan debugging should generate a build plan 1`] = `
```

Comment:
When reading this, I'm not really sure what I'm looking at. There should at least be some extensive documentation on how to read the output from this flag.

Reply:
I added comments in the code in [b693a40](https://github.com/microsoft/rushstack/pull/4660/commits/b693a4093f45c98c1ca903ffbd63c1570c60e9fd) and [77bdaa0](https://github.com/microsoft/rushstack/pull/4660/commits/77bdaa078e464415197f2be49737aa7e688c75aa). Where should I add full documentation?

Reply:
@octogonz can help you post it to https://rushjs.io/